selfdrive/ui/translations/main_ja.ts

```diff
@@ -556,17 +556,17 @@ location set</source>
     <message>
         <location filename="../qt/text.cc" line="41"/>
         <source>Exit</source>
-        <translation>退出</translation>
+        <translation>閉じる</translation>
     </message>
     <message>
         <location filename="../qt/util.cc" line="18"/>
         <source>dashcam</source>
-        <translation>ダッシュカム</translation>
+        <translation>ﾄﾞﾗｲﾌﾞﾚｺｰﾀﾞｰ</translation>
```

Comment:
"drive recorder" seems to be a more commonly used phrase for dashcam, open to reverting.